docs/api_changes.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: default
+title: API changes
+nav_order: 6
+---
+
+# API changes
+
+This page aims to detail specific changes to the UDA client API, as well as describing how deprecations will be managed.
+
+
+## Notable release history
+
+| UDA version | Description of API changes |
+|:------------|:---------------------------|
+| 2.7.0 | Some accidental breaking api changes are introduced, including some name changes from idam to uda, and the introduction of a new `client_flags` argument to some functions |
+| 2.7.6 | A legacy mapping header is introduced which reintroduces the syntax that was lost in 2.7.0, while keeping the option to use new updated names (such as udaFree instead of idamFree) |
+| (planned) 3.0.0 | Removal of all references to legacy "idam" name in all api functions. Imposing consitent naming convention in all API functions. Legacy API mapping header to be updated to permit continued use of old names for a more forgiving deprecation of old syntax |
+
+
+## The legacy api headers
+
+Where old functions names are deprecated the old syntax may still be available in a header file mapping the old syntax to the new. This will allow codes to continue to build against newer UDA library versions until they are ready to update. 
+
+The naming convention for these header files is they mirror the existsing headers they correspond to. For example to create a name mapping for a function which is declared in the `client.h` file, such as `udaFree`, the new header will be called `legacy_client.h`. To make use of a previous API function signature that is being replaced, client code simply needs to include the corresponding legacy header. 
```

Comment:
> "The naming convention for these header files is they mirror the existing headers"

Doesn't quite make sense, maybe 'header files is **that** they', I don't know, just doesn't sound right